sormas-api/src/main/resources/strings.properties

```diff
@@ -751,7 +760,7 @@ headingEventGroupUnlinkEventIssue = Issue while unlinking events to event group
 headingExportUserRightsFailed = Export user rights failed
 headingExternalMessageDownload = Download message
 headingNoExternalMessagesSelected = No messages selected
-headingExternalMessagesDeleteProcessed = Delete processed message(s)
+headingExternalMessagesEligibleForDeletion = Delete processed message(s)
```

Comment:
What's the reason for this renaming?

Reply:
I've thought that it would be more consistent if the heading, the message and in the code appears the same naming with eligible.

Now (after my changes) we have like this in the notification:
headingExternalMessagesEligibleForDeletion = Delete processed message(s)
messageExternalMessagesEligibleForDeletion = Only unprocessed messages can be deleted
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/370d009e-2425-4842-8cbc-113b958ddc02)


And we display the notification from here:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/0b3b02b7-4b96-4b4f-9f97-62258e4d94ad)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/547d0bbb-4f46-4f33-a7fa-af5d749c550a)

If you want, I can change back the namings to headingExternalMessagesDeleteProcessed and 
messageExternalMessagesDeleteProcessed.

Reply:
I think I would just remove the heading from that popup and simply display the error message; the heading doesn't really provide anything meaningful anyway